src/components/Comments/CommentForm.tsx

```diff
@@ -0,0 +1,72 @@
+import { FC, useCallback, useRef } from "react";
+import { Block, Elem } from "../../utils/bem";
+import { ReactComponent as IconSend } from "../../assets/icons/send.svg";
+
+import "./CommentForm.styl";
+import { TextArea } from "../../common/TextArea/TextArea";
+import { observer } from "mobx-react";
+
+
+export type CommentFormProps = {
+  commentStore: any,
+  value?: string,
+  onChange?: (value: string) => void,
+  onInput?: (value: string) => void,
+  inline?: boolean,
+  rows?: number,
+  maxRows?: number,
+}
+
+export const CommentForm: FC<CommentFormProps> = observer(({
+  commentStore,
+  value = "", 
+  inline = true,
+  onChange,
+  onInput,
+  rows = 1,
+  maxRows = 4,
+}) => {
+  const formRef = useRef<HTMLFormElement>(null);
+  const actionRef = useRef<{ update?: (text?: string) => void }>({});
+
+  const onSubmit = useCallback(async (e?: any) => {
+    e?.preventDefault?.();
+
+    if (!formRef.current) return;
+    
+    const comment = new FormData(formRef.current).get("comment") as string;
+
+    if (!comment.trim()) return;
+
+    try {
+      actionRef.current.update?.("");
+
+      await commentStore.addComment(comment);
+      
+    } catch(err) {
+      actionRef.current.update?.(comment || "");
+      console.log(err);
```

Comment:
small suggestion since we're intending to log the error in console
```suggestion
      console.error(err);
```

Reply:
this is a good thought, should implement app wide